addons/pharma/functions/fnc_treatmentAdvanced_AmiodaroneLocal.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+/*
+ * Author: Mazinski.H, Edited by MiszczuZPolski
+ * Applies Bradycardia randomly
+ *
+ * Arguments:
+ * 0: Patient <OBJECT>
+ *
+ * Return Value:
+ * Succesful treatment started <BOOL>
```

Comment:
where is the return value? is a return Value needed? If not only change the comment :)